resources/logfs/dl.css

```diff
@@ -27,14 +28,28 @@ button:first-of-type {
   background: #000;
   color: #fff;
 }
+tr:nth-child(even) {
+  background: #f2f2f2;
+}
```

Comment:
Incidental fix: this was in the wrong order vs the hover effect so the hover effect only applied to odd rows.